pagopa/pn-frontend
# Pn 2428   pn 2354

## Short description
Setup Cypress and added some tests for PA.

## List of Features
- Setup of Cypress
- Added Cypress tests about auth, notification filters and validation process for new notification.

## How to test
Do a refresh monorepo because of new packages added then run `yarn cypress open`
Please check [this](https://pagopa.atlassian.net/wiki/spaces/PN/pages/531664452/Configurazioni) for configuration.
